src/test/ui/resolve/issue-14254.stderr

```diff
@@ -2,7 +2,7 @@ error[E0425]: cannot find function `baz` in this scope
   --> $DIR/issue-14254.rs:29:9
    |
 29 |         baz();
-   |         ^^^ did you mean `self.baz(...)`?
+   |         ^^^ help: did you mean `self.baz`
```

Comment:
The issue is that we have no idea whether the suggestion author wrote `try` or `did you mean`, so we can't generate a questionmark out of thin air. We could cover the common cases like `did you mean`, but `have you tried` won't get caught or any other textual changes